Update the RDS restore docs post-replatforming (#4500)
alphagov/govuk-developer-docs: source/manual/howto-backup-and-restore-in-aws-rds.html.md

This command will wait until the database is ready, and then exit without any output.

### 6. Get the new database's hostname

Once the database is ready, fetch its hostname:

```
gds-cli aws govuk-<environment>-admin aws rds describe-db-instances --db-instance-identifier "restored-${db_instance_identifier}" --query 'DBInstances[].Endpoint.Address'
```

### 7. Make a change to the database contents

> Through the app's user interface, or via the app console, make a change that you
> can use as a sense check to verify that the database switch has been successful.

Reviewer: ```suggestion
Check that the app can still read and write to its database.
```

> For example, you might create a draft edition of something, or modify or delete
> a record. After switching to the restored database, your changes should be undone.

Reviewer: Best to avoid choices in playbooks where possible.

```suggestion
For example, you might create a draft edition and use the preview feature to check that the app saved your changes.
```

In this example we open a Rails console on local-links-manager, from which we can

> make a change such as deleting an old record:

Reviewer: This step seems a bit vague. Not sure if I've correctly understood the intent but it sounds like it's about checking that the client app can read/write the restored database? If so, can we replace it with a more specific and straightforward check?

Could maybe delete this paragraph and the Rails console snippet and just leave the bit about about checking that the app can still write stuff by e.g. creating a draft edition.

Author: I think the intent was to make sure the restore was successful by making a change before the pre-restore, and then checking if the change was reverted post-restore.

It's probably easier/less confusing if I remove that whole section and suggest checking which DB ActiveRecord is connected to via the Rails console instead.
